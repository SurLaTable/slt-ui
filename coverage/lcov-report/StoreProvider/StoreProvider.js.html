<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for StoreProvider/StoreProvider.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">StoreProvider</a> StoreProvider.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>4/4</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>4/4</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">10x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import PropTypes from 'prop-types';
import { Provider } from 'react-redux';
import configureStore from '../utils/storeTools';
&nbsp;
export const store = configureStore();
&nbsp;
const StoreProvider = (props) =&gt; (
	&lt;Provider store={store}&gt;
		&lt;React.Fragment&gt;{props.children}&lt;/React.Fragment&gt;
	&lt;/Provider&gt;
);
&nbsp;
StoreProvider.propTypes = {
	children: PropTypes.node
};
&nbsp;
export default StoreProvider;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Wed Sep 12 2018 09:29:34 GMT-0800 (GMT-08:00)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
