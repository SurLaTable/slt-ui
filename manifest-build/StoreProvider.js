(window.webpackJsonp=window.webpackJsonp||[]).push([["StoreProvider"],{"./manifest/temp/slt/StoreProvider.js":function(e,o,r){"use strict";o.__esModule=!0,o.default=void 0;var s=r("./src/ComparisonChart/index.js");o.default=s.StoreProvider}}]);