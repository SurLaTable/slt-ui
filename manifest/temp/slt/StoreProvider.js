
            export {StoreProvider as default} from '../../../src/ComparisonChart/index.js';
          