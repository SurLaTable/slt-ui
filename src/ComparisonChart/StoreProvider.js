import React from 'react';
// import { render } from 'react-dom';
import { Provider } from 'react-redux';
import { createStore } from 'redux';

import sltReducers from './reducers/sltReducers';

const sltStore = createStore(
    sltReducers,
    // Enable DevTools, switch to localStorage on Prod:
    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__()
);

const StoreProvider = (props) => (
    <Provider store={sltStore}>
			<React.Fragment>{props.children}</React.Fragment>
    </Provider>
);
export default StoreProvider;
