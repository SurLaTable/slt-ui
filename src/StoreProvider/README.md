```
<div data-component="StoreProvider"></div>
```
