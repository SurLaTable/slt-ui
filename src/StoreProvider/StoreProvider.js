import React from 'react';
import PropTypes from 'prop-types';
import { Provider } from 'react-redux';
import configureStore from '../utils/storeTools';

export const store = configureStore();

const StoreProvider = (props) => (
	<Provider store={store}>
		<React.Fragment>{props.children}</React.Fragment>
	</Provider>
);

StoreProvider.propTypes = {
	children: PropTypes.node
};

export default StoreProvider;
