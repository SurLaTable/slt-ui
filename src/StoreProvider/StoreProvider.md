```jsx
<StoreProvider />
```

```html
<div data-component="StoreProvider"></div>
```
