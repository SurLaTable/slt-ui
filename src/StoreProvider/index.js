import React from 'react';
import { Provider } from 'react-redux';
import configureStore from '../utils/storeTools';

export const store = configureStore();
const StoreProvider = (props) => (
	<Provider store={store}>
		<React.Fragment>{props.children}</React.Fragment>
	</Provider>
);
export default StoreProvider;
