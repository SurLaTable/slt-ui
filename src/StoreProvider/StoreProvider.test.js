import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import StoreProvider from './StoreProvider';

test('Render', () => {
	const renderer = new ShallowRenderer();
	renderer.render(<StoreProvider />);
	const result = renderer.getRenderOutput();
	expect(result).toMatchSnapshot();
});
