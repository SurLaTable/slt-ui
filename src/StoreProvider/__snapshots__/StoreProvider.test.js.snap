// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render 1`] = `
<Provider
  store={
    Object {
      "asyncReducers": Object {},
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <React.Fragment />
</Provider>
`;
